cypress/integration/seededFlows/articleFlows/followFromArticleLiquidTag.spec.js

```diff
@@ -0,0 +1,47 @@
+describe('Follow from article liquid tag', () => {
+  beforeEach(() => {
```

Comment:
Tests (blocking): add a test for `Follow Back` text when a user followed a user and that user goes to follow them.